mantidproject/mantid
# Removed call to unused property in fit

Removed the refrences to OutputNIterations property in fit as it was no longer needed. In master running fit in a GUI will get stuck with a progress bar at 100%. This should fix that problem. 

**To test:**
Fit to some data using the fitting GUI and make sure that the progress bar disappears on completion. 
Fit to some data using the multiple fitting GUI and make sure it completes and the progress bar disappears. 

Fixes #19993

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
